go/vt/vttablet/tabletmanager/rpc_replication.go

```diff
@@ -619,13 +632,16 @@ func (tm *TabletManager) setMasterLocked(ctx context.Context, parentAlias *topod
 				return err
 			}
 		}
+		// Clear replication sentinel flag for this replica
+		tm.replManager.setReplicationStopped(false)
```

Comment:
Can you please explain why setting to `false`? It seems to me like if anything we should set to `true`, because we're replicating up to a certain point, then stopping, correct?

Reply:
The logic here is that "false" means that "replication is no longer stopped on purpose;  please restart if necessary";  while "true" means "replication is stopped on purpose, please do not restart".

Reply:
Got it! Thanks for clarifying, the function name is confusing.